assessments/unit-01/code.js

```diff
@@ -5,3 +5,107 @@ const emit = (a, b, product) => {
 };
 
 // Write your code here ...
+const averageWeight = (totalWeight, numItems) => {
+  return (totalWeight/numItems)
+}
+
+const hypotenuse = (a, b) => {
+  return Math.sqrt(a**2 + b**2)
+}
+
+const maxRadius = (widthOfArea, heightOfArea) => {
+  return (Math.min (widthOfArea, heightOfArea))/2
+}
+
+const numCircles = (radius, widthOfArea) => {
+  return Math.floor (widthOfArea/radius)
+}
+
+const offset = (widthOfArea, widthOfFigure) => {
+  return (widthOfArea-((Math.floor(widthOfArea/widthOfFigure))*widthOfFigure))/2
+}
+
+const canSleepIn = (isWeekday, onVacation) => {
+  return !isWeekday || onVacation
+}
+
+const canGoToProm = (isASenior, invitedBySenior, onExclusionList) => {
+  return isASenior || invitedBySenior &&! onExclusionList
+}
+
+const getsSpeedingTicket = (mph, isCopGrouchy) => {
+  if (isCopGrouchy === true) {
+    if (mph > 65) {
+      return true
+    } else {
+      return false
+    }
+  } else {
+    if (mph > 70) {
+      return true
+    } else {
+      return false
+    }
+  }
+}
+
+const moreThanTwiceAsLong = (a, b) => {
+  if (a.length > (b.length)*2) {
+    return true
+  } else {
+    return false
+  }
+}
+
+const aFartherThanB = (a, b, c) => {
+  if (Math.abs (a-c) > Math.abs (b-c)) {
+    return true
+  } else {
+    return false
+  }
+}
+
+const firstHalf = (s) => {
+  return s.substring (0, Math.floor(s.length/2))
```

Comment:
Good but formatting should be:

```suggestion
  return s.substring(0, Math.floor(s.length / 2))
```